internal/pipeline/pipeline.go

```diff
@@ -228,6 +248,68 @@ func (f *Function) Validate() error {
 	return nil
 }
 
+// runner returns a fnRunner from the image and configs of
+// this function.
+func (f *Function) runner() (kio.Filter, error) {
+	config, err := f.config()
+	if err != nil {
+		return nil, err
+	}
+	return &fnRunner{
```

Comment:
I think it will be better to look at f.Image to see if it uses `set-annotator`, then return the buildin annotator function. That way we can avoid hardcoding. WDYT ?

Reply:
If we want to use a special image name then I think it's better to make it more distinct, like `builtin-set-annotator`.